tests/org.jboss.tools.fuse.ui.bot.test/src/org/jboss/tools/fuse/ui/bot/test/CamelVersionDetectionTest.java

```diff
@@ -0,0 +1,69 @@
+package org.jboss.tools.fuse.ui.bot.test;
+
+import static org.jboss.reddeer.requirements.server.ServerReqState.PRESENT;
+import static org.jboss.tools.runtime.reddeer.requirement.ServerReqType.Fuse;
+import static org.junit.Assert.assertTrue;
+
+import org.jboss.reddeer.core.handler.ShellHandler;
+import org.jboss.reddeer.junit.requirement.inject.InjectRequirement;
+import org.jboss.reddeer.junit.runner.RedDeerSuite;
+import org.jboss.tools.common.reddeer.view.ErrorLogView;
+import org.jboss.tools.fuse.reddeer.wizard.NewFuseIntegrationProjectWizard;
+import org.jboss.tools.fuse.ui.bot.test.utils.ProjectFactory;
+import org.jboss.tools.runtime.reddeer.impl.ServerFuse;
+import org.jboss.tools.runtime.reddeer.requirement.ServerRequirement;
+import org.jboss.tools.runtime.reddeer.requirement.ServerRequirement.Server;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Tests wizard for creating a new Fuse Integration Project
+ * 
+ * @author djelinek
+ */
+@RunWith(RedDeerSuite.class)
+@Server(type = Fuse, state = PRESENT)
+public class CamelVersionDetectionTest {
+	
+	@InjectRequirement
+	private ServerRequirement serverRequirement;
+
+	/**
+	 * Clean test environment
+	 */
+	@After
```

Comment:
It would be nice to also execute this method before the test (just to be sure there is no project called 'test').